generators/server/templates/src/main/resources/config/liquibase/master.xml.ejs

```diff
@@ -64,6 +64,14 @@
     <property name="uuidType" value="uuid" dbms="mssql"/>
     <property name="datetimeType" value="datetime" dbms="mssql"/>
 <%_ } _%>
+    <property name="clobType" value="clob" dbms="mysql, oracle, mssql, mariadb, postgresql"/>
+<%_ if (prodDatabaseTypeMysql || prodDatabaseTypeMariadb) { _%>
+    <property name="uuidType" value="varchar(36)" dbms="h2, mysql, mariadb"/>
+<%_ } else if (!prodDatabaseTypeNeo4j) { _%>
+    <property name="uuidType" value="uuid" dbms="h2, <%= prodDatabaseType %>"/>
+<%_ } _%>
+    <property name="datetimeType" value="datetime(6)" dbms="mysql, mariadb"/>
+    <property name="datetimeType" value="datetime" dbms="oracle, mssql, postgresql, h2"/>
```

Comment:
@fbiville With the latest changes done in liquibase, we currently define properties by database type.
You need so to define now the values for neo4j explicitly